.changeset/add-package-licenses.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Prettier is complaining about the formatting of the changeset file. If you can run prettier on that real quick then I’ll merge. Thanks!

Reply:
and probably a quick rebase against main as well, thanks!

Reply:
done!

Reply:
Hmmm, it looks like it pulled in everything that's changed since then. Looks like you either need to rebase against upstream/main or fetch upstream into your local main and rebase off of that. fwiw I make this mistake all the time.

Reply:
try `git rebase -i origin/main` and delete everything from the list except the two commits you intend to include. That should fix it.

Reply:
Also remember you need to force push the result, that is an intentional result of rebase in this scenario.